src/main/java/com/example/bookstoreapp/controller/OrderController.java

```diff
@@ -66,7 +69,8 @@ public OrderItemDto getOrderItem(@PathVariable Long orderId, @PathVariable Long
 
     @GetMapping("{orderId}/items")
     @PreAuthorize("hasRole('USER')")
-    @Operation(summary = "Get order item", description = "Get order item")
+    @Operation(summary = "Get order", description = "Get order by Id if order belongs to current "
+            + "User")
     @ResponseStatus(HttpStatus.OK)
     public OrderDto getOrder(@PathVariable Long orderId) {
         return orderService.getOrder(orderId);
```

Comment:
same here, we should ensure that user can access only his own orders/cart/cartItems/orderItems